Userland/Libraries/LibGfx/JPGLoader.cpp

```diff
@@ -1238,17 +1237,14 @@ RefPtr<Gfx::Bitmap> load_jpg(String const& path)
     auto file_or_error = MappedFile::map(path);
     if (file_or_error.is_error())
         return nullptr;
-    auto bitmap = load_jpg_impl((const u8*)file_or_error.value()->data(), file_or_error.value()->size());
-    if (bitmap)
-        bitmap->set_mmap_name(String::formatted("Gfx::Bitmap [{}] - Decoded JPG: {}", bitmap->size(), LexicalPath::canonicalized_path(path)));
-    return bitmap;
+    return load_jpg_from_memory((const u8*)file_or_error.value()->data(), file_or_error.value()->size(), path);
 }
 
-RefPtr<Gfx::Bitmap> load_jpg_from_memory(const u8* data, size_t length)
+RefPtr<Gfx::Bitmap> load_jpg_from_memory(const u8* data, size_t length, String const& mmap_name)
```

Comment:
if you're touching this anyway, turn `const u8*` to `u8 const*` (and other lines you've already modified)

Reply:
Sounds good, done for all loaders.